libcloud/compute/drivers/dimensiondata.py

```diff
@@ -1690,6 +1691,12 @@ def _to_node(self, element):
             if has_network_info else \
             element.find(fixxpath('nic', TYPES_URN)).get('privateIpv4')
 
+        node.extra['ipv6'] = element.find(
```

Comment:
Correct! This should read 
    extra['ipv6'} = ... 

New commit to follow, sorry for the dirty copy and paste.